examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -99,6 +101,14 @@
                   {
                     "name": "JENKINS_PASSWORD",
                     "value": "${JENKINS_PASSWORD}"
+                  },
+                  {
+                    "name": "KUBERNETES_MASTER",
```

Comment:
well, technically you could just change the run command in the pod without changing the image, but not sure it's worth the trouble.